spec/jobs/open_order_cycle_job_spec.rb

```diff
@@ -75,6 +75,30 @@
     end
   end
 
+  context "with cloned order cycle" do
+    subject { OpenOrderCycleJob.perform_now(cloned_order_cycle.id) }
+
+    let!(:cloned_order_cycle) do
+      order_cycle.update!(opened_at: now - 5.minutes)
+
+      coc = OrderCycles::CloneService.new(order_cycle.reload).create
+      coc.update!(orders_open_at: now + 5.minutes)
+      coc.reload
+
+      coc
+    end
+
+    it "marks as open" do
+      expect {
+        subject
+        cloned_order_cycle.reload
+      }
+        .to change { cloned_order_cycle.opened_at }
+
+      expect(cloned_order_cycle.opened_at).to be_within(1).of(now)
+    end
+  end
```

Comment:
I think we should also check that `OrderCycles::WebhookService` is actually called , see the "enqueues webhook job" example at line 24 for inspiration.